### ISSUE TITLE
minimize, maximize and exit icons not displayed

### ISSUE THREAD

**Issue by elevenhsoft** [pop-os/libcosmic#550]
Hi

Not sure when this happened. Just updated dependencies in my app including libcosmic and noticed on GNOME, Exit, Minimize and Maximize icons are not displayed. They are blank. Just empty ;p

It's happening when Cosmic icons theme is not installed, so installed Cosmic icons to my $HOME/.local/share/icons dir and icons are back.

**Comment by mmstick** [comment]
the cosmic icon theme is required

**Comment by elevenhsoft** [comment]
Ok. Before it was working so my though was some regression. Thanks!

**Comment by elevenhsoft** [comment]
Well, I have to reopen this issue. The problem is with cross desktop support now.
Since latest changes, and cosmic icon theme requirement. Apps no longer properly works on different desktops and distros. I don't think that's a good idea to follow cosmic icon theme. And what about flatpaks?

**Comment by mmstick** [comment]
You need to update libcosmic to pull in https://github.com/pop-os/libcosmic/commit/716b3dee560d92cb4b651de10daeee5c1b64818e

In general, you should bundle the cosmic icon theme with flatpaks.

**Comment by elevenhsoft** [comment]
Yeah, I saw that commit. Unfortunately it working only on other OS than Linux. On Linux there still is issue, when we don't have cosmic icon theme installed, we don't see that icons. It's essential for working app, only 3 icons. Can't we use this icons also on Linux? I'm talking about this commit. If we use cached icons for other OS why Linux must have different behavior. As I understand for better consistency, but maybe some kind of fallback? For now, we have broken apps if we use latest libcosmic.

**Comment by mmstick** [comment]
You need to get the icon theme installed on Linux

**Comment by Drakulix** [comment]
Btw same is the case for libadwaita, which requires the Adwaita icon theme. This shouldn't be an issue for application developers, but simply handled as dependencies of whatever package system they are using. For flatpaks we'll probably want a cosmic-runtime as a base for libcosmic apps at some point, which should simply include the icon theme.

**Comment by elevenhsoft** [comment]
Oh, did not know that. It's bad either ^^ But it is how it is.

> For flatpaks we'll probably want a cosmic-runtime as a base for libcosmic apps at some point, which should simply include the icon theme.

That would be awesome to see.

**Comment by wiiznokes** [comment]
We could also use other installed themes right? I believe hicolor is the fallback
